package.json

```diff
@@ -71,6 +71,7 @@
     "eslint": "^8.5.0",
     "eslint-config-prettier": "^6.10.1",
     "extract-zip": "^2.0.1",
+    "html-minifier-terser": "^7.0.0-beta.0",
```

Comment:
This should be in `dependencies`, not `devDependencies` -- and the old `html-minifier` module needs to be removed from the dependencies as well.